### PR TITLE
update teacher dashboard, code studio homepage, and /educate/k5 with …

### PR DESCRIPTION
**Pull request by tanyaparker**
…unplugged curriculum

**teacher dashboard**
image

image

**Code Studio homepage**

(signed in)
image

(signed out)
image

**code.org/educate/k5**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
I'll update the i18n gsheet with the changed strings.

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -474,6 +474,11 @@ en:
     minecraft:
       title: "Minecraft"
       body: "Explore a Minecraft world through code."
+    unplugged:
+      title: "Unplugged Lessons"
+      audience: "Ages 6+"
+      body: "We've compiled a list of all of our unplugged lessons for you to use in your classroom. Now you can teach the fundamentals of computer science, whether you have computers in your classroom or not! Try using these lessons as a stand alone course or as complementary lessons for any computer science course."
```
Comment (breville) [inline dashboard/config/locales/en.yml]: I think stand alone might be better as stand-alone.

File: pegasus/cache/i18n/en-US.yml [hunk 2]
```diff
@@ -659,9 +659,8 @@
   dashboard_faq_k5: "K-5 Frequently Asked Questions"
   dashboard_announcements: "Announcements"
   dashboard_announce_view_all: "View all"
-  dashboard_announce_message: "<b>New feature</b> - Now you can move your students between your sections or to another teacher's section."
-  dashboard_announce_message_second: "<b>Phase one applications closing</b> - Do you (or a teacher you know) want to teach high school computer science? Professional learning workshops are open this summer for a full year intro course that students love. And it's designed so you can teach without any prior experience. Join our free workshops by signing up before April 22nd."
-  dashboard_announce_button_text: "Learn more"
+  dashboard_announce_message: "<b>April 22, 2016</b> - Unplugged activities are a vital part of the learning process, but teachers tell us that finding all of the resources can be confusing and time-consuming. We listened! Now, you can find all of the unplugged resources on a single landing page. Associate the lessons with any computer science class, or use stand alone as an independent course."
```
Comment (breville) [inline pegasus/cache/i18n/en-US.yml]: should we say "our unplugged" (as we do elsewhere) rather than "the unplugged"?

**Comment by breville** [comment]
lgtm, thanks for the screenshots.

File: dashboard/config/locales/en.yml (hunk 1 above)
Comment (tanyaparker) [inline dashboard/config/locales/en.yml]: Definitely

File: pegasus/cache/i18n/en-US.yml (hunk 2 above)
Comment (tanyaparker) [inline pegasus/cache/i18n/en-US.yml]: They're not all ours though (the safety ones)

Comment (tanyaparker) [inline pegasus/cache/i18n/en-US.yml]: Maybe I should update the other places to remove "our" ?

Comment (breville) [inline pegasus/cache/i18n/en-US.yml]: i feel like people expect courses listed on studio to be "our" but if not, then maybe good to be consistent.

**Pull request merged**